tests/cases/fourslash/getOccurrencesDeclare1.ts

```diff
@@ -44,7 +44,6 @@
 ////            protected prot1;
 ////
 ////            protected constructor(public public, protected protected, private private) {
-////                public = private = protected;

```

Comment:
With that line, we consider the below declare to be containing in different scope,the above declare will be in module m scope while the below declare will not be considered in module m scope.